osx/KA-Lite/KA-Lite/AppDelegate.m

```diff
@@ -766,6 +778,17 @@ - (IBAction)start:(id)sender {
 }
 
 
+- (IBAction)startButton:(id)sender {
```

Comment:
@amodia Why duplicate the same code as above?  You must re-use or create a re-usable function instead of doing a copy-paste.